hnn_core/gui/gui.py

```diff
@@ -0,0 +1,1008 @@
+                   description='MPI cmd:', disabled=False)
+
+    joblib_cores = BoundedIntText(value=1, min=1,
+                                  max=multiprocessing.cpu_count(),
+                                  description='Cores:', disabled=False)
+
+    backend_config = Output()
+    handle_backend_change(backend_selection.value, backend_config, mpi_cmd,
+                          joblib_cores)
+
+    simulation_box = VBox([tstop, tstep, ntrials, backend_selection,
+                           backend_config])
+
+    # Sliders to change local-connectivity params
+    sliders = [
+        _get_sliders(params, [
+            'gbar_L2Pyr_L2Pyr_ampa', 'gbar_L2Pyr_L2Pyr_nmda',
+            'gbar_L2Basket_L2Pyr_gabaa', 'gbar_L2Basket_L2Pyr_gabab'
+        ]),
+        _get_sliders(params, [
+            'gbar_L2Pyr_L5Pyr', 'gbar_L2Basket_L5Pyr', 'gbar_L5Pyr_L5Pyr_ampa',
+            'gbar_L5Pyr_L5Pyr_nmda', 'gbar_L5Basket_L5Pyr_gabaa',
+            'gbar_L5Basket_L5Pyr_gabab'
+        ]),
+        _get_sliders(params,
+                     ['gbar_L2Pyr_L2Basket', 'gbar_L2Basket_L2Basket']),
+        _get_sliders(params, [
+            'gbar_L2Pyr_L5Basket', 'gbar_L5Pyr_L5Basket',
+            'gbar_L5Basket_L5Basket'
+        ])
+    ]
+
+    # accordians to group local-connectivity by cell type
+    boxes = [VBox(slider) for slider in sliders]
+    titles = [
+        'Layer 2/3 Pyramidal', 'Layer 5 Pyramidal', 'Layer 2 Basket',
+        'Layer 5 Basket'
+    ]
+    cell_connectivity = Accordion(children=boxes)
+    for idx, title in enumerate(titles):
+        cell_connectivity.set_title(idx, title)
+
+    # Dropdown for different drives
+    layout = Layout(width='200px', height='100px')
+
+    drive_type_selection = RadioButtons(
+        options=['Evoked', 'Poisson', 'Rhythmic'],
+        value='Evoked',
+        description='Drive:',
+        disabled=False,
+        layout=layout)
+
+    location_selection = RadioButtons(options=['proximal', 'distal'],
+                                      value='proximal',
+                                      description='Location',
+                                      disabled=False,
+                                      layout=layout)
+
+    add_drive_button = create_expanded_button('Add drive', 'primary',
+                                              height='30px')
+
+    drive_selections = VBox(
+        [HBox([drive_type_selection, location_selection]), add_drive_button])
+
+    # XXX: should be simpler to use Stacked class starting
+    # from IPywidgets > 8.0
+    drives_options = VBox([drive_selections, drives_out])
+
+    # Tabs for left pane
+    left_tab = Tab()
+    left_tab.children = [simulation_box, cell_connectivity, drives_options]
+    titles = ['Simulation', 'Cell connectivity', 'Drives']
+    for idx, title in enumerate(titles):
+        left_tab.set_title(idx, title)
+
+    # Running status
+    simulation_status = Label(value="Not running")
```

Comment:
I really like this! But do you think we could also have a background color? Maybe using the `HTML` widget? Green when simulation is finished, yellow when it is running